File: Sprint2 - Back/Api/ConexãoVagas.DbFirst/Domains/Candidato.cs
using System;
using System.Collections.Generic;

namespace ConexaoVagasAPI.Domains
{
    public partial class Candidato
    {
        public Candidato()
        {
            Candidatura = new HashSet<Candidatura>();
            HabilidadeCandidato = new HashSet<HabilidadeCandidato>();
        }

        public int IdUsuario { get; set; }
        public int IdStatusUsuario { get; set; }
        public string Curso { get; set; }
        public string Matricula { get; set; }
        public string Cpf { get; set; }
        public string Rg { get; set; }
        public string Cep { get; set; }
        public string EnderecoCandidato { get; set; }
        public int NumeroEndereco { get; set; }
        public string TelefoneCandidato { get; set; }
        public string CelularCandidato { get; set; }
        public DateTime DataNascimento { get; set; }
        public int Visualizacao { get; set; }

        public StatusUsuario IdStatusUsuarioNavigation { get; set; }
        public Usuario IdUsuarioNavigation { get; set; }
        public ICollection<Candidatura> Candidatura { get; set; }
        public ICollection<HabilidadeCandidato> HabilidadeCandidato { get; set; }
    }
}

File: Sprint3 - Front/React.Js/ConexaoVagas/ConexaoVagas-Front/src/interfaces/candidato.ts
import { Candidatura } from "./candidatura";
import { Endereco } from "./endereco";
import { HabilidadeCandidato } from "./habilidadeCandidato";
import { StatusUsuario } from "./statusUsuario";
import { Usuario } from "./usuario";

export interface Candidato {
    idUsuario: number;
    idStatusUsuario: number;
    idEndereco: number | null;
    nome: string;
    sobrenome: string;
    curso: string;
    matricula: string;
    cpf: string;
    rg: string;
    cep: string;
    enderecoCandidato: string;
    numeroEndereco: number;
    telefoneCandidato: string;
    celularCandidato: string;
    dataNascimento: string;
    visualizacao: number;
    idEnderecoNavigation: Endereco;
    idStatusUsuarioNavigation: StatusUsuario;
    idUsuarioNavigation: Usuario;
    candidatura: Candidatura[];
    habilidadeCandidato: HabilidadeCandidato[];
}